### ISSUE TITLE
calc: encryption for backups

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#2483]
also check backup default filenames

also optionally in non-calc version

**Comment by tuomas2** [comment]
Or not necessarily only for calc, but for AB too. User-selectable password/passphrase as a encryption key.

**Comment by Noitarud** [comment]
Friendly reminder
![password_strength](https://github.com/AndBible/and-bible/assets/98794719/ae2ffa23-b169-4e43-b2ae-8847ba0be9db)

**Comment by tuomas2** [comment]
~@Noitarud that applies for online services. AndBible is offline app. Nobody would try to bruteforce even simple 4-digit pin by tapping one by one.~

**Comment by Noitarud** [comment]
No, they would use their own program to bruteforce the files, looking for catchwords… in that case you have no chance. You need only something simple to discourage the casual observer. A four digit pin can be forced. You just allocate some time each day to do ten or twenty and you'll get it. I was thinking that for my car stereo - 36 per day, I will have it before 36 days. And the three digit pin on the back of your bank card? Disable it NOW. Even with mine disabled the bank still gave away what mine was (I got blockage alerts).

**Comment by tuomas2** [comment]
My bad. Of course applies to encrypted backup file passphrase. Somehow in my mind was just pin code used in calc.

**Comment by tuomas2** [comment]
Estimate 5h
